packages/governance/src/votes/votes.cairo

```diff
@@ -253,61 +281,21 @@ pub mod ERC20VotesComponent {
             to: ContractAddress,
             amount: u256
         ) {
-            let zero_address = Zero::zero();
             let block_timestamp = starknet::get_block_timestamp();
-            if (from == zero_address) {
-                let mut trace = self.ERC20Votes_total_checkpoints.read();
-                trace.push(block_timestamp, trace.latest() + amount);
+            if from.is_zero() {
+                let mut trace = self.Votes_total_checkpoints.deref();
+                trace.push(block_timestamp, trace.into().latest() + amount);
             }
-            if (to == zero_address) {
-                let mut trace = self.ERC20Votes_total_checkpoints.read();
-                trace.push(block_timestamp, trace.latest() - amount);
+            if to.is_zero() {
+                let mut trace = self.Votes_total_checkpoints.deref();
+                trace.push(block_timestamp, trace.into().latest() - amount);
             }
             self.move_delegate_votes(self.delegates(from), self.delegates(to), amount);
         }
-
-        /// Returns the number of checkpoints for `account`.
-        fn num_checkpoints(self: @ComponentState<TContractState>, account: ContractAddress) -> u32 {
-            self.ERC20Votes_delegate_checkpoints.read(account).length()
-        }
-
-        /// Returns the `pos`-th checkpoint for `account`.
-        fn checkpoints(
-            self: @ComponentState<TContractState>, account: ContractAddress, pos: u32
-        ) -> Checkpoint {
-            self.ERC20Votes_delegate_checkpoints.read(account).at(pos)
-        }
```

Comment:
Why remove these two from the internal trait?

Reply:
Not really, thanks for catching it. They got lost when moving/refactoring the component. I just added both and tests for them